sources/Common/src/Options.cpp

```diff
@@ -110,10 +112,21 @@ Options::parse(int argc, char* argv[]) -> std::tuple<bool, Request> {
     if (vm.count("log-level") > 0) {
       config_.dynawoLogLevel = vm["log-level"].as<ParsedLogLevel>().logLevelDefinition;
     }
-    return std::forward_as_tuple(true, Request::RUN_SIMULATION);
+
+    if (vm.count("flat") > 0) {
+      if (vm.count("contingencies") > 0) {
+        return Request::RUN_SIMULATION_NSA;
+      } else {
+        return Request::ERROR;
```

Comment:
I'm not sure the error message will be clear enough in this case, maybe we should add in the help that contingency file is required when nsa option is used